src/index.ts

```diff
@@ -2,7 +2,7 @@ import * as core from "@actions/core";
 
 import { PassThrough, Stream } from "stream";
 import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
-import { Session, cloudApi, serviceClients } from "@yandex-cloud/nodejs-sdk";
+import { Session, cloudApi, serviceClients, waitForOperation } from "@yandex-cloud/nodejs-sdk";
```

Comment:
It sound great! But what if we would add a parameter for action to enable/disable it? We can have it enabled by default, and in case someone wants just having disabled will able to do it.

Reply:
I didn't get the idea of making error handling optional 🤔
Now action ends successfully even if the function was not created. So, you can never be sure whether the function was created or not until you go to the Yandex cloud console and check the result manually. 
That behavior seems confusing for me and I already spend some time trying to understand why the deployment was successful, but I don't see any changes when I run the deployed function. =)

Reply:
I see, I got it now, was thinking it's just a wait for operation to finish